recipes/premake/5.x/patches/0001-5.0.0-alpha15-mingw.patch

```diff
@@ -1,25 +1,37 @@
+--- a/contrib/curl/lib/select.h
++++ b/contrib/curl/lib/select.h
+@@ -36,7 +36,8 @@
+
+ #if !defined(HAVE_STRUCT_POLLFD) && \
+     !defined(HAVE_SYS_POLL_H) && \
+-    !defined(HAVE_POLL_H)
++    !defined(HAVE_POLL_H)  && \
++    !defined(POLLIN)
+
+ #define POLLIN      0x01
+ #define POLLPRI     0x02
 --- src/host/os_isdir.c
 +++ src/host/os_isdir.c
 @@ -9,7 +9,7 @@
  #include "premake.h"
- 
```

Comment:
Aren't these trailing spaces importing in the patches?

Reply:
Well, they are strange, because that changes are not visible locally

Reply:
Spaces change removed

Reply:
It's probably a setting in your ide.
The root `.editorconfig` of this repo has a rule to not remove trailing spaced from patches.